src/dsp_tools/utils/stack_handling.py

```diff
@@ -255,39 +295,46 @@ def _start_docker_containers(self) -> None:
         self._start_remaining_docker_containers()
         self._execute_docker_system_prune()
 
-    def start_stack(
-        self,
-        max_file_size: Optional[int] = None,
-        enforce_docker_system_prune: bool = False,
-        suppress_docker_system_prune: bool = False,
-    ) -> bool:
+    def _inject_latest_tag_into_docker_compose_file(self) -> None:
         """
-        Start the Docker containers of DSP-API and DSP-APP, and load some basic data models and data. After startup, ask
-        user if Docker should be pruned or not.
+        Replace the version of the DSP-API Docker image in docker-compose.yml with "latest".
 
-        Args:
-            max_file_size: max. multimedia file size allowed by SIPI, in MB (max: 100'000)
-            enforce_docker_system_prune: if True, prune Docker without asking the user
-            suppress_docker_system_prune: if True, don't prune Docker (and don't ask)
+        Raises:
+            BaseError: if it is not possible to inject the "latest" tag into docker-compose.yml
+        """
+        docker_compose_file = self.__docker_path_of_user / "docker-compose.yml"
+        if not docker_compose_file.is_file():
+            raise BaseError(
+                "Cannot inject the 'latest' tag into docker-compose.yml, "
+                f"because there is no such file in {self.__docker_path_of_user}"
+            )
+
+        with open(docker_compose_file, "r", encoding="utf-8") as f:
+            docker_compose_text = yaml.safe_load(f)
+        old_tag = docker_compose_text.get("services", {}).get("api", {}).get("image", "")
+        if not re.search(r"daschswiss/knora-api:\d+\.\d+\.\d+", old_tag):
+            raise BaseError(f"Invalid tag '{old_tag}' in docker-compose.yml > services > api > image")
+
+        docker_compose_text["services"]["api"]["image"] = "daschswiss/knora-api:latest"
+        with open(docker_compose_file, "w", encoding="utf-8") as f:
+            yaml.dump(docker_compose_text, f)
```

Comment:
This is one of the relevant code changes

Reply:
or you could use an environment variable to control the version: https://stackoverflow.com/a/62706577/11914562